arrays/arrays.c

```diff
@@ -85,7 +85,7 @@ char *arr_read(Array *arr, int index) {
 
   // Throw an error if the index is greater than the current count
   if(index >= arr->count){
-    printf("Capacity has been reached");
+    printf("Array capacity has been reached");
     return 0;
```

Comment:
This might throw an error because this function is set up to output void (i.e. doesn't output anything). Here you're outputting an integer.